qiita_ware/util.py

```diff
@@ -97,38 +96,6 @@ def per_sample_sequences(iter_, max_seqs, min_seqs=1, random_buf_size=100000):
             yield (sequence_id, sequence)
 
 
-def stats_from_df(df):
```

Comment:
That sentence is actually contradictory, since you are removing the function now. I would suggest either remove both this function and the code or leave both, but not remove one but not the other. Note that this is creating an import error in the test file, so the other tests on that file do not run, which are independent to your changes. There is no need to break more tests than the strictly needed ones.